public/pages/DetectorsList/utils/constants.ts

```diff
@@ -18,3 +18,5 @@ export enum DETECTOR_ACTION {
   STOP,
   DELETE,
 }
+
+export const singleDetectorErrorMsg = 'Not Found';
```

Comment:
For constant variable, name is usually in upper case

Reply:
Good catch, will change.